src/assets/scripts/client/airport/Arrival/ArrivalCyclic.js

```diff
@@ -1,91 +0,0 @@
-/* eslint-disable camelcase, no-underscore-dangle, no-mixed-operators, func-names, object-shorthand */
-import _random from 'lodash/random';
-import ArrivalBase from './ArrivalBase';
-import { convertMinutesToSeconds } from '../../utilities/unitConverters';
-import { TIME } from '../../constants/globalConstants';
-
-/**
- * Generate arrivals in cyclic pattern
- * Arrival rate varies as pictured below. Rate at which the arrival rate
- * increases or decreases remains constant throughout the cycle.
-
- * |---o---------------o---------------o---------------o-----------| < - - - - - - max arrival rate
- * | o   o           o   o           o   o           o   o         |   +variation
- * o-------o-------o-------o-------o-------o-------o-------o-------o < - - - - - - avg arrival rate
- * |         o   o |         o   o           o   o           o   o |   -variation
- * |-----------o---|-----------o---------------o---------------o---| < - - - - - - min arrival rate
- * |<---period---->|           |<---period---->|
```

Comment:
So glad you called that out!  I want to preserve them, for sure!  I'm not sure where, yet, but they should remain.